packages/unlock-assets/package.json

```diff
@@ -5,11 +5,11 @@
   "author": "unlock inc",
   "license": "MIT",
   "main": "dist/index.js",
-  "module": "dist/index.modern.js",
-  "source": "src/index.js",
+  "module": "dist/index.mjs",
```

Comment:
Use `exports` instead of module defined on the node official documentation site.